R/formatters_var_labels.R

```diff
@@ -43,7 +43,7 @@ col_labels <- function(x, fill = FALSE) {
     return(character(0L))
   }
 
-  labels <- lapply(x, attr, "label")
+  labels <- lapply(x, function(x) as.vector(attr(x, "label")[[1]]))
```

Comment:
Why `[[1]]`?

Reply:
This is to continue in case label is of length greater than 1 - e.g. `attr(x, "label") <- letters`.
There are few options available out of which these two have the most of sense to me: either hard stop or take the first (and throw a warning maybe?)

Reply:
Why stop there?
```suggestion
labels <- lapply(x, attr, "label")
lapply(labels, checkmate::assert_string, .var.name = "column label")
lapply(lapply(labels, names), checkmate::assert_names, type = "unnamed", .var.name = "column label")
```

There used to be a check https://github.com/insightsengineering/teal.data/pull/299.

Reply:
assertion has been added